lib/joystick.dart

```diff
@@ -0,0 +1,7 @@
+export 'src/components/joystick/joystick_action.dart';
+export 'src/components/joystick/joystick_component.dart';
```

Comment:
Why is this its own export and not together with the other components?

Reply:
I like this to group, then the components can just export one joystick file